src/manual/Installing/Community-projects.md

```diff
@@ -178,6 +178,15 @@ Simple ClamAV REST proxy. Builds on top of [clamav-java](#clamav-java) which is
 
 [Get clamav-rest](https://github.com/solita/clamav-rest)
 
+#### Go
+
+##### go-clamav
```

Comment:
```suggestion
##### go-clamav | clamd
```

This sounds like a clamd client library. Let's put it under `## ClamAV Bindings` (below), and change this line to show that these bindings interact with `clamd`.